tests/integration/test_fim/test_registry/test_multiple_registries/common.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import win32con
```

Comment:
Instead of importing the entire library, import only the used variables
```suggestion
from win32con import REG_SZ
```